### PR TITLE
Querypack equivalent of ossec rootkit db

### PR DESCRIPTION
**Pull request by lambda-conjecture**
Querypack generated with tool at https://github.com/Uptycs/ossec-to-osquery

### CODE REVIEW

**Comment by jedi22** [comment]
Are all the rootkits linux only (none apply to darwin systems)?

**Comment by lambda-conjecture** [comment]
"Are all the rootkits linux only (none apply to darwin systems)?"

Good question. As far as I know, they are linux only (search for "OS X" in the doc below...

https://media.readthedocs.org/pdf/ossec-docs/stable/ossec-docs.pdf

**Comment by jedi22** [comment]
@poppyseedplehzr I see no problems merging this, this would be a query pack not included in our osquery.conf example though (which loads all the others in that folder). Should we also add it there? Even if it's only on linux?

**Comment by jedi22** [comment]
@lambda-conjecture I connected with @theopolis and we decided it should be added to the osquery.conf example just like everything else. Once that's done, we can merge.

**Comment by lambda-conjecture** [comment]
Hello, I updated tools/osquery.example.conf and also rename ossec_rootkit.conf to ossec-rootkit.conf to be consistent with the naming of the other query packs.

File: tools/deployment/osquery.example.conf [hunk 1]
```diff
@@ -75,5 +75,6 @@
     // "osx-attacks": "/usr/share/osquery/packs/osx-attacks.conf",
     // "vuln-management": "/usr/share/osquery/packs/vuln-management.conf",
     // "hardware-monitoring": "/usr/share/osquery/packs/hardware-monitoring.conf"
```
Comment (jedi22) [inline tools/deployment/osquery.example.conf]: Small thing, this needs a comma at the end

Comment (lambda-conjecture) [inline tools/deployment/osquery.example.conf]: Fixed - thanks for catching it.

**Pull request merged**